### DISCUSSION TITLE
Window::request_animation_frame with callback with an argument?

### DISCUSSION THREAD

**Discussion by g-braeunlich** [wasm-bindgen/wasm-bindgen#4012] [Q&A] [answered]
Hello guys.
The [request animation frame example](https://rustwasm.github.io/wasm-bindgen/examples/request-animation-frame.html) shows how to use a closure with no arguments. Is it also possible to pass a closure with 1 argument, i.e. the timestamp argument as shown [here](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame#examples)?

**Comment by daxpedda** [comment, marked as answer]
Indeed, you are free to define parameters yourself in `Closure`s.

In this case `Closure<dyn FnMut(f64)>` would be appropriate.
